packages/create/src/cli/initialize.ts/findPositionalFrom.ts

```diff
@@ -0,0 +1,11 @@
+import { isLocalPath } from "../utils.js";
+
+export function findPositionalFrom(positionals: string[]) {
+	// TODO: intelligently skip past node.js, etc.?
```

Comment:
TODO: add test coverage around cases like...

* pnpm vs yarn vs npm
* `npx` vs `node .../bin/index.js` (and the equivalents in other packages)